creator-node/src/dbManager.js

```diff
@@ -432,6 +432,61 @@ class DBManager {
       (indexCreation) => indexCreation.relname === 'Files_storagePath_idx'
     )?.length
   }
+
+  static async _getLegacyStoragePathsAndCids(cursor, batchSize, dir) {
+    const queryResult = await models.File.findAll({
+      attributes: ['storagePath', 'multihash'],
+      where: {
+        multihash: { [sequelize.Op.gte]: cursor },
+        type: {
+          [dir ? sequelize.Op.eq : sequelize.Op.ne]: models.File.Types.dir
+        },
+        storagePath: {
+          [sequelize.Op.notLike]: '/file_storage/files/%',
+          [sequelize.Op.like]: '/file_storage/%'
```

Comment:
@SidSethi do you think it's okay to use this hardcoded path here? I'm thinking legacy can only ever use /file_storage/<CID> so as long as the query has `LIKE '/file_storage%' AND NOT LIKE '/file_storage/files/%'` then it should be good?

Reply:
yeah i think fine